app/components/publications/publication-status-selector.js

```diff
@@ -2,59 +2,18 @@ import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
-
 export default class PublicationStatusSelector extends Component {
   @service store;
-
   @tracked publicationStatusses
-  @tracked statusOptions = [];
 
   constructor() {
     super(...arguments);
     this.publicationStatusses = this.store.peekAll('publication-status').sortBy('position');
-    this.fillStatusOptions();
-  }
-
-  fillStatusOptions() {
-    for (const publicationStatus of this.publicationStatusses) {
-      let icon;
-      if (publicationStatus.isPending) {
-        icon =  {
-          svg: 'clock',
-          color: 'warning',
-        };
-      } else if (publicationStatus.isPublished) {
-        icon =   {
-          svg: 'circle-check',
-          color: 'success',
-        };
-      } else if (publicationStatus.isPaused) {
-        icon =  {
-          svg: 'circle-pause',
-          color: 'muted',
-        };
-      } else if (publicationStatus.isWithdrawn) {
-        icon =   {
-          svg: 'circle-error',
-          color: 'danger',
-        };
-      } else {
-        icon =  {
-          color: 'muted',
-        };
-      }
-      const option = {
-        id: publicationStatus.id,
-        label: publicationStatus.label,
-        icon: icon,
-      };
-      this.statusOptions.push(option);
-    }
   }
 
   get publicationStatus() {
```

Comment:
Nu de plain-javascript-voorstelling van de codelijst hier weg is, wordt deze getter ook overbodig, bedenk ik me nu. Je kan in de template rechtstreeks `@publicationStatus` gebruiken (wat er uit deze getter komt is via een omweg exact hetzelfde object).